packages/buffetjs-core/src/components/Toggle/index.js

```diff
@@ -11,17 +11,32 @@ import { Toggle as StyledToggle, ToggleWrapper } from '@buffetjs/styles';
 import Label from '../Label';
 
 function Toggle({ disabled, id, className, name, onChange, value }) {
+  const handleRef = element => {
```

Comment:
Can you wrap this in a `useCallback` hook to avoid the ref being computed on every render?